src/Components/Dropdowns/Dropdown.scss

```diff
@@ -52,6 +53,8 @@ $dropdown-item--padding-v: $cf-space-2xs;
   flex-direction: column;
   align-items: stretch;
   cursor: pointer;
+  margin-right: 2px;
+  margin-left: 1px;
```

Comment:
These two stick out, but they're just to please the `absolute` positioned, Scrollbar library we're using

Reply:
Can this be solved using `position` and `left` and `right` rather than with using margins? Semantically, there's a big difference between position and margin, and we should use positioning for laying things out, and margins for when we need speec around an element. I'm not super familiar with how this is used, though

Reply:
Got you! I'll try that!

Reply:
<img width="311" alt="Screen Shot 2022-10-18 at 8 19 39 AM" src="https://user-images.githubusercontent.com/18511823/196472653-bce006b3-dcd1-48d7-9781-ef02fa1233ac.png">

It still cuts off the right end of it, I think this is what is happening:

![IMG_D4759C8952C6-1](https://user-images.githubusercontent.com/18511823/196473654-a3f92211-966c-459a-9dcb-98c66def39d5.jpeg)

So, when I add the margin, the border falls under the available space.

Unfortunately, I am not able to change the width of the inner wrapper, since it's controlled by the react-custom-scrollbar library.

Reply:
This was always a problem, but we never noticed because we didn't use `borders` as an indicator for being selected. Now that we use the border, we see that the 1px of the right is cut off.

Reply:
is it causing problems to have the border? julia and i also discussed maybe not needing a background color at all on selected items. we included it because that's what we have now—adjusted to be lower contrast so it's less distracting—but i think the selected state for the radio buttons and checkboxes has enough contrast that we might not need the background color+border as well. let me know what you think. @ChitlangeSahas @hoorayimhelping @juliajames12